examples/medplum-photon-integration/src/bots/handle-prescription-event.ts

```diff
@@ -64,6 +67,32 @@ export async function handler(medplum: MedplumClient, event: BotEvent<PhotonWebh
   return prescription;
 }
 
+async function getMedicationCodeFromPrescription(
+  prescriptionId: string,
+  authToken: string
+): Promise<Record<string, string | undefined>> {
+  const query = `
+    query prescription($id: ID!) {
+      prescription(id: $id) {
+        treatment {
+          name
+          codes {
+            rxcui
+          }
+        }
+      }
+    }
+  `;
+
+  const variables = { id: prescriptionId };
+
+  const body = JSON.stringify({ query, variables });
+  const result = await photonGraphqlFetch(body, authToken);
+  const rxcui = result.data.prescription.treatment.codes.rxcui;
```

Comment:
Add null guards to avoid errors
```suggestion
  const rxcui = result.data?.prescription?.treatment?.codes?.rxcui;
```